Adding sample code to Azure.Communication.NetworkTraversal

ajpeacock0 opened the pull request:
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

ajpeacock0 committed:
Updating links and date in README and CHANGELOG

DominikMe commented:
@ajpeacock0 Writing samples for .NET is a little more involved. The snippet references in the .md files need to be backed by corresponding #region areas in a test .cs file. Then you need to run a command that will copy over the regions into the .md files. More info here and examples in the other packages (e.g. Communication.Identity)
https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets

ajpeacock0 commented:
> @ajpeacock0 Writing samples for .NET is a little more involved. The snippet references in the .md files need to be backed by corresponding #region areas in a test .cs file. Then you need to run a command that will copy over the regions into the .md files. More info here and examples in the other packages (e.g. Communication.Identity)
> https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets

I ran `.\Update-Snippets.ps1 communication` and it appeared to work on the samples code in this PR

```
Replaced Snippet:CreateCommunicationRelayClient in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClient.md
Replaced Snippet:CreateCommunicationRelayFromAccessKey in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClient.md
Replaced Snippet:CreateCommunicationRelayFromToken in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClient.md
Replaced Snippet:GetRelayConfigurationAsync in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClient.md
Replaced Snippet:CreateCommunicationRelayClient in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClientAsync.md
Replaced Snippet:CreateCommunicationRelayFromAccessKey in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClientAsync.md
Replaced Snippet:CreateCommunicationRelayFromToken in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClientAsync.md
Replaced Snippet:GetRelayConfigurationAsync in C:\Users\anpea\git_repos\azure-sdk-for-net-ajpeacock0\sdk\communication\Azure.Communication.NetworkTraversal\samples\Sample1_CommunicationRelayClientAsync.md
```

However there were no code changes. Is there something wrong you've noticed in my current PR?
